src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -759,7 +759,12 @@ namespace AppInstaller::CLI::Workflow
 
         if (table.IsEmpty())
         {
-            context.Reporter.Info() << Resource::String::NoInstalledPackageFound << std::endl;
+            if (m_onlyShowUpgrades) {
+                context.Reporter.Info() << Resource::String::UpdateNotApplicable << std::endl;

```

Comment:
I think we still want to differentiate "app not installed" vs "update not applicable". This change seems ok with winget upgrade --all, but we want clearer message for winget upgrade <specific package>

Reply:
Got it. I restored the original message, but at least removed the duplication. What do you think?